pimcore/lib/Pimcore/Bundle/AdminBundle/Resources/config/services.yml

```diff
@@ -49,3 +49,10 @@ services:
     pimcore.object.gridcolumconfig:
         class: Pimcore\Model\DataObject\GridColumnConfig\Service
         public: true
+
+    #
+    # IMPORT CONFIG
+    #
+    pimcore.object.importconfig:
```

Comment:
Can this one be defined as fully qualified classname instead (Symfony 3.3+ format)?